build/_k-restore.shade

```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
```

Comment:
Lots of other "k" stuff in here (including filenames) - do we have a work item for these?